coins.yml

```diff
@@ -0,0 +1,212 @@
+- id: 60
+  index: 0x8000003c
+  symbol: ETH
+  trust: ethereum
+  name: Ether
+  link: https://ethereum.org/ether
+  decimals: 18
+  sample: '0xfc10cab6a50a1ab10c56983c80cc82afc6559cf1'
```

Comment:
did you consider to rename to `sampleAddress` ? we might do more samples in the future